src/ase/cw/model/Server.java

```diff
@@ -47,31 +55,33 @@ public int getId() {
         return serverId;
     }
 
-    @Override
-    public void setBusy(boolean busy) {
-        this.busy = busy;
+    private void setStatus(status status) {
+        serverStatus = status;
+        ssl.onServerStatusChange(this);
```

Comment:
please make a check:
if(status != serverStatus){

}

otherwise ssl.onServerStatusChange(this) will be called twice if something like this happens:
setStatus(pause);
setStatus(pause);
But the serverStatus did only change one time

Reply:
good point. thanks